_posts/2022-12-01-cockpit-281.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Cockpit 281
+author: garrett
+date: '2022-12-01'
+tags: cockpit, machines
+slug: cockpit-281
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 281 and cockpit-machines 279:
+
+
+## Add style switcher
+
+Due to browsers not always picking up the correct system dark mode and users preferring to override the system dark mode Cockpit now has a style switcher in the session menu.
+
+![Screenshot from 2022-12-01 15-09-39]({{ site.baseurl }}/images/281-add-style-switcher.png)
+
+## Disallow root login by default
+On all operating systems, logging in as root with a username/password via SSH has been disallowed for a long time. Cockpit now also disallows it by default on new installations, but still allows it on upgrades for backwards compatibility. You can configure this in `/etc/cockpit/disallowed-users`.
+
+## Autodetect image type when adding a disk via a custom path
```

Comment:
Missing 'Machines' header prefix. We used to always add the prefix for all pages. Isn't this the case now?

Reply:
It is. It's supposed to be autodetected and applied to the PR's title, but when someone sets a custom header, that's used instead (so it has to be manually added to manual headers).

Fixed in the PR; updated in the notes here.